lib/utils.js

```diff
@@ -30,8 +31,20 @@ function normalizeBinary (binaryPath, platform, arch) {
                /linux/i.test(platform) ? "linux" + arch :
                platform;
 
+    var channelNames = ["firefox", "firefoxdeveloperedition", "beta", "nightly", "aurora"];
+
     var app = binaryPath.toLowerCase();
-    binaryPath = normalizeBinary.paths[app + " on " + platform] ||
+    var result = null;
+    if (platform === "osx" && channelNames.indexOf(binaryPath) !== -1) {
+      // Try to find the app for this channel.
+      // mdfind "kMDItemCFBundleIdentifier == 'org.mozilla.firefoxdeveloperedition'"
+      var results = spawnSync("mdfind", ["kMDItemCFBundleIdentifier == 'org.mozilla." + binaryPath + "'"]);
```

Comment:
I remember once I disabled md indexing to speed up my machine :) I think that was a couple major versions ago of OS X though. Not that it matters since this is just a fallback.